validator/cycle/process_tasks.py

```diff
@@ -253,7 +316,34 @@ async def _handle_delayed_tasks(config: Config):
     await asyncio.gather(*[_move_back_to_looking_for_nodes(task, config) for task in finished_delay_tasks])
 
 
+async def _move_to_preevaluation_status(task, config):
+    task.status = TaskStatus.PREEVALUATION
+    logger.info(f"Moving this one back {task}")
```

Comment:
back to where? 

Moving {task} from {status} to {status} if you wnata. log. shouldn't this have the extra context in?